libs/atqr/domain/src/lib/domain-spec/stories/visitorAdoptsGoal.disl

```diff
@@ -0,0 +1,32 @@
+read_also api.disl
+read_also enums.disl
+read_also ui.disl
+
+domain ATQR
+  purpose "A platform for people to challenge each other to accomplish goals"
+
+context Challenges
+
+  root Goal
+    @goal: string
+    @challenges: Challenge[]
+    @pursuers: returns @challenges.map(c => c.enrollments.map(e => e.player)).flat()
+
+    CreateChallenge?
```

Comment:
I don't understand why there's this empty command here

Reply:
I think the idea was lost btwn changes, can't remember why I left this command behind.